core/drivers/pl061_gpio.c

```diff
@@ -156,6 +191,8 @@ static const struct gpio_ops pl061_ops = {
 	.set_direction	= pl061_set_direction,
 	.get_value	= pl061_get_value,
 	.set_value	= pl061_set_value,
+	.get_interrupt = pl061_get_interrupt,
```

Comment:
The '=' don't quite match the above lines. The previous lines use a tab, and this is a single space. Suggest using a tab for consistency, even though it really shouldn't be a tab.